connect/proxy/proxy.go

```diff
@@ -109,6 +114,11 @@ func (p *Proxy) Serve() error {
 					continue
 				}
 
+				if existing := active[uc.String()]; existing != nil {
+					p.logger.Info("Re-using existing listener", "listener", uc.String())
```

Comment:
Does this need to be logged?

Reply:
I don't think the user cares. They just want their changes to take effect and this is an optimization that we're reusing it vs shutting it all down and starting it back up.